src/apolloClient.js

```diff
@@ -1,12 +1,39 @@
 import { ApolloClient } from 'apollo-client'
 import { ApolloLink } from 'apollo-link'
-
 import apolloLogger from 'apollo-link-logger'
 import { SchemaLink } from 'apollo-link-schema'
+import { onError } from 'apollo-link-error'
 import { InMemoryCache } from 'apollo-cache-inmemory'
 import schema from 'graphql-server'
 
+const errorLink = onError(({ graphQLErrors, networkError, response }) => {
+  if (networkError) {
+    console.log(`[Network error]: ${networkError}`)
+    response.errors.hposConnection = false
```

Comment:
This variable name is confusing.

`response.errors.hposConnection` reads to me as "is there an error in the hposConnection?" not "do we have a working hposConnection?"

Ideally we would put it somewhere else in the response object, and rename it to something like:

`response.isHposConnectionActive`

Would have to dig into the docs to see if that would be possible. But even just renaming it

`response.errors.isHposConnectionActive`

would make it much clearer.